Makefile

```diff
@@ -15,13 +15,20 @@ prodceleryworkers:
 devserver:
 	yarn run devserver
 
+test:
+	cd contentcuration; pytest
+
+endtoendtest:
+	# launch all studio's dependent services using docker-compose, and then run the tests
+	docker-compose run studio-app make test
```

Comment:
You can replicate the travis test environment by running this command on your machine!